content/doc/upgrade-guide/2.107.adoc

```diff
@@ -60,10 +60,18 @@ It is no longer necessary to provide a CSRF crumb when sending an HTTP request w
 
 This is a list of confirmed regressions introduced in this release.
 
+// TODO: post known regressions for all releases?
+// JENKINS-48770 - introduced in 2.89
 // JENKINS-48821 is not needed here as it was introduced in 2.89.x due to backport
+// JENKINS-49588 - unconfirmed, maybe introduced in 2.107
 * link:https://issues.jenkins-ci.org/browse/JENKINS-49392[JENKINS-49392]:
   Some job configuration forms may fail to load if plugin:violations[Violations Plugin] is installed.
 
+In addition to the list above,
```

Comment:
Please keep the file formatting consistent with the rest of the file, which only breaks on sentence boundaries.